frontend/src/app/view-claims/view-claims.component.html

```diff
@@ -49,7 +49,7 @@ <h3 [lyTyp]="'title'">{{ claim.vehicle.vin }}</h3>
   <!--STATUS # 3-->
   <ly-tab>
     <button ly-tab-label (click)="sortClaims(3)">Claim Resolved</button>
-    <ly-grid *ngIf="sortClaims(3).length > 0; else noClaims" container [spacing]="16">
+    <ly-grid *ngIf="sortClaims(3).length > 0; then hasClaims else noClaims" container [spacing]="16">
```

Comment:
Same here - what is this doing?

Reply:
That was something I was trying out and forgot to delete